c2corg_ui/static/menu-discourse/menu.html

```diff
@@ -0,0 +1,117 @@
+## HEAD
+<link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css" rel="stylesheet">
+
+## En-tete
+<aside id="sidemenu">
+  <div class="page-menu">
+    <a href="//www.demov6.camptocamp.org">
+      <img src="http://www.demov6.camptocamp.org/static/4c851e7/img/logo.svg" alt="Camptocamp.org" class="logo"/>
+    </a>
+    <div class="menu-open-btn">
+      <i class="glyphicon glyphicon-align-justify"></i>
+    </div>
+  </div>
+  <ul class="nav">
+    <li class="main-menu">
+      <a href="//www.demov6.camptocamp.org">
+        <i class="glyphicon glyphicon-home"></i>
+        <span class="lang en menu-text">Home</span>
+        <span class="lang fr menu-text">Accueil</span>
```

Comment:
How does the i18n work here? Only the current lang span is shown?
Is it the system suggested by @gberaudo in the initial demo menu?
I guess using angular-gettext is too complicated here?

If we stick to this system, translations for the other languages are needed.